### PR TITLE
Add media Queries for mobiles

### PR DESCRIPTION
**Pull request by mittalprince**
@rishabhc32 review pr

### CODE REVIEW

**Comment by mittalprince** [comment]
@rishabhc32 in this pr i move all inline css into static/style.css and add media querry for icons

File: themes/material/layouts/_default/baseof.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <link href="https://fonts.googleapis.com/css?family=Noto+Sans|Source+Code+Pro:300" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel=”stylesheet” media=”screen and (min-width:320px) and (max-width:700px) “href=”css/yourstylesheet.css” />
```
Comment (rishabhc32) [inline themes/material/layouts/_default/baseof.html]: What is the use of this line?

**Comment by rishabhc32** [comment]
There are still icon alignment issues. See in this video.
https://streamable.com/2mos3

**Comment by rishabhc32** [comment]
Please add media query for all devices first.
If it doesn't work then you can look into moving icons next to name.

**Comment by mittalprince** [comment]
@rishabhc32 sir please review changes.
thanks

**Comment by mittalprince** [comment]
icons
@rishabhc32 I think this might me best option for us.
thanks

**Pull request merged**